packages/turf-transform-scale/index.js

```diff
@@ -15,7 +15,7 @@ import { getCoord, getCoords, getType } from "@turf/invariant";
  *
  * @name transformScale
  * @param {GeoJSON} geojson GeoJSON to be scaled
- * @param {number} factor of scaling, positive or negative values greater than 0
+ * @param {number} factor of scaling, positive values greater than 0
```

Comment:
Perhaps we can add a note that setting the factor to 1 will return the current size, so setting to between 0 & 1 will make the geometry smaller, and setting to above 1 will return a larger geometry.